packages/libs/react-ui/src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  boldTextClass,
+  imageClass,
+  imageContainerClass,
+  linkClass,
+  linkContainerClass,
+  profileCardClass,
+  tagClass,
+  tagContainerClass,
+} from './ProfileCard.css';
+
+import { Grid } from '@components/Grid';
+import { Tag } from '@components/Tag';
+import React, { FC } from 'react';
+
+export default {};
+export interface IProfileCardProps {
+  name: string;
+  title: string;
+  tags: string[];
+  imageSrc: string;
+  links: Record<string, string>;
```

Comment:
I think using configuration for this is fine since it's quite simple and links will be external, however I do think tags and links should be optional